girder_worker/app.py

```diff
@@ -130,10 +132,30 @@ def call_item_task(self, inputs, outputs={}):
         return self.run.call_item_task(inputs, outputs)
 
 
-@before_task_publish.connect
+@before_task_publish.connect # noqa
 def girder_before_task_publish(sender=None, body=None, exchange=None,
                                routing_key=None, headers=None, properties=None,
                                declare=None, retry_policy=None, **kwargs):
+
+    def _setArgToChainedTasks(arg, val):
```

Comment:
eventually we will want this to be much more sophisticated (e.g. with chords and other chaining)  but this is probably fine for this PR